Makefile

```diff
@@ -20,6 +20,8 @@ TINYUSB_CDC=1
 # Now we're all set to include gossamer's make rules.
 include $(GOSSAMER_PATH)/make.mk
 
+CFLAGS+=-D_POSIX_C_SOURCE=200112L
```

Comment:
This lets us use strtok_r in newlib (otherwise --std=c99 in gossamer means we're restricted to standard C functions)